### PR TITLE
Take first IP address found (usually IPv4)

### PR DESCRIPTION
**Pull request by Bischoff**
Minimal fix for issue #805

Instead of using the last IP address met, at the risk of taking an useless IPv6 link-local address, take the first one met.

Before approving and merging this, please read issue #805 for a discussion of the problem, and other possible fixes.

### CODE REVIEW

**Review by zhshuj — COMMENTED** [review]
The code changes LGTM. 
Perhaps the UT cases need be updated accordingly too.

**Review by zhshuj — APPROVED** [review]

**Comment by zhshuj** [comment]
@jichenjc @bjhuangr Can you help review?

File: zvmsdk/smtclient.py [hunk 1]
```diff
@@ -364,19 +364,22 @@ def get_adapters_info(self, userid):
                 lan_name = line.strip().split('=')[-1]
                 adapter['lan_name'] = lan_name
             if 'mac_address=' in line and not found_mac:
-                mac_addr = line.strip().split('=')[-1]
-                pattern = re.compile('.{2}')
-                mac_address = ':'.join(pattern.findall(mac_addr))
-                adapter['mac_address'] = mac_address
+                if not found_mac:
```
Comment (bjhuangr) [inline zvmsdk/smtclient.py:367]: duplicate condition with line 366?

Comment (Bischoff) [reply zvmsdk/smtclient.py:367]: Good catch. Fixing it.

Comment (Bischoff) [reply zvmsdk/smtclient.py:367]: Done

Comment (Bischoff) [reply zvmsdk/smtclient.py:367]: And retested. Works.

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**